### PR TITLE
update Czech locale

### PR DESCRIPTION
**Pull request by PetrDlouhy**
Czech locale update. Please update also on Transifex.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -15,7 +15,6 @@
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
-  enable_extension "postgis"
```
Comment (nikolai-b) [inline db/schema.rb]: Could you remove this?

**Comment by nikolai-b** [comment]
Looks good, thanks @PetrDlouhy 
If you can remove the schema change then I'll merge it in

**Comment by PetrDlouhy** [comment]
The schema change was my mistake, now it is removed.

**Comment by PetrDlouhy** [comment]
Is the Circleci problem in this PR?

**Comment by PetrDlouhy** [comment]
Hi @nikolai-b can this PR be pulled, or is there any problem? :-)
Thanks

**Comment by PetrDlouhy** [comment]
Hello, is there anything I could help with to get this pulled?

**Comment by nikolai-b** [comment]
Sorry I missed this.

**Pull request merged**